pyqtgraph/multiprocess/bootstrap.py

```diff
@@ -25,9 +25,12 @@
 
     pyqtapis = opts.pop('pyqtapis', None)
     if pyqtapis is not None:
-        import sip
-        for k,v in pyqtapis.items():
-            sip.setapi(k, v)
+        try:
+            from PyQt5 import sip
+        except ImportError:
+            import sip
+            for k,v in pyqtapis.items():
```

Comment:
Should this go in a "finally" clause?

Reply:
Hello, according to the Debian maintainer:
"Also, calls like "sip.setapi('QDate', 2)" are not needed at all with PyQt5.
They were needed only with PyQt4 and Python 2. It is safe to delete them."

This is why I'm looping on the "ImportError" case, because its where the old api is being used.

Reply:
BTW I'm not a Python-savvy man, so please double check my statement :D